components/Team/TeamWorkList.tsx

```diff
@@ -1,42 +1,123 @@
 import { observer } from 'mobx-react';
-import { Ratio, Image, Accordion } from 'react-bootstrap';
+
+import Link from 'next/link';
+import {
+  Ratio,
+  Image,
+  Row,
+  Col,
+  Card,
+  Accordion,
+  Button,
+} from 'react-bootstrap';
 
 import { ScrollListProps, ScrollList } from '../ScrollList';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCalendarDay } from '@fortawesome/free-solid-svg-icons';
+
 import { TeamWork, TeamWorkType } from '../../models/Team';
 import activityStore from '../../models/Activity';
 
 export interface TeamWorkListProps extends ScrollListProps<TeamWork> {
   activity: string;
   team: string;
+  size?: 'sm' | 'lg';
+  onDelete?: (id: any) => void;
 }
 
 @observer
 export class TeamWorkList extends ScrollList<TeamWorkListProps> {
   store = activityStore.teamOf(this.props.activity).workOf(this.props.team);
-
-  static Layout = ({ value = [] }: TeamWorkListProps) => (
+  extraProps: Partial<TeamWorkListProps> = {
+    onDelete: (id: any) => {
+      if (!id) return;
+      this.store.deleteOne(id);
+    },
+  };
+  static Layout = ({
+    value = [],
+    size,
+    onDelete,
+    activity,
+    team,
+  }: TeamWorkListProps) => (
     <Accordion>
-      {value.map(({ updatedAt, id, title, description, type, url }) => (
-        <Accordion.Item eventKey={id!} key={id}>
-          <Accordion.Header>
-            {title} - {updatedAt ? updatedAt.slice(0, 10) + ' 更新' : ''}
-          </Accordion.Header>
-          <Accordion.Body>
-            <p>{description}</p>
-            {type === TeamWorkType.IMAGE ? (
-              <Image src={url} className="mw-100" alt={title} />
-            ) : type === TeamWorkType.VIDEO ? (
-              <Ratio aspectRatio="16x9">
-                <video controls width="250" src={url} />
-              </Ratio>
-            ) : (
-              <a href={url} title={title}>
-                {title}
-              </a>
-            )}
-          </Accordion.Body>
-        </Accordion.Item>
-      ))}
+      <Link href={`/activity/${activity}/team/${team}/work/create`}>
+        <Button variant="success" className="me-3 mb-2">
```

Comment:
```suggestion
      <Link href={`/activity/${activity}/team/${team}/work/create`} passHref>
        <Button variant="success" className="me-3 mb-2">
```
如此可只生成一层 `<a />`，而非 `<a><button /></a>`